go.mod

```diff
@@ -73,7 +73,7 @@ require (
 	github.com/cespare/xxhash/v2 v2.1.1 // indirect
 	github.com/coreos/go-semver v0.3.0 // indirect
 	github.com/coreos/go-systemd/v22 v22.3.2 // indirect
-	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
+	github.com/cpuguy83/go-md2man/v2 v2.0.2 // indirect
```

Comment:
I tested this PR by trying to reproduce the diff by bumping just the CLI library. I don't get this version bump for `go-md2man` and also not for `blackfriday`. Are you sure `go mod tidy` did bump those?

Reply:
I ran `go get -u github.com/urfave/cli@v1.22.9` and got those updates.

Reply:
Ah, you're right. I only updated the version in the `go.mod` file and ran `go mod tidy`.

Reply:
I'll update with `go mod tidy` right away.

Reply:
Yeah, that'd be great. Would like to keep the diff in dependencies as small as possible. Also looking at the `go.mod` of the `urfave/cli` library at the `v1.22.9` version doesn't look like it actually requires these two other libraries to be bumped. So not sure what `go get -u` does here.

Reply:
Ok. I'll just bump cli then.